src/serde/deserialize_program.rs

```diff
@@ -55,9 +56,22 @@ impl Default for ApTracking {
     }
 }
 
-#[derive(Deserialize, Debug)]
+#[derive(Deserialize, Debug, PartialEq, Clone)]
 pub struct Identifier {
     pub pc: Option<usize>,
+    #[serde(rename(deserialize = "type"))]
+    pub type_: Option<String>,
+    #[serde(default)]
+    #[serde(deserialize_with = "bigint_from_number")]
+    pub value: Option<BigInt>,
+}
+
+fn bigint_from_number<'de, D>(deserializer: D) -> Result<Option<BigInt>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let n = Number::deserialize(deserializer)?;
+    return Ok(Some(bigint_str!(n.to_string().as_bytes())));
```

Comment:
The macro `bigint_str` uses unwraps and should only be used in tests, I would recommend using `BigInt::parse_bytes()` and handling the error case

Reply:
Thanks @fmoletta !
[Done](https://github.com/lambdaclass/cairo-rs/pull/422/commits/6d84bd49d9d635fe36b346ca52b59a610dacdea0)
I didn't add any error handling because in the case the value from the JSON is not a number the` let n = Number::deserialize(deserializer)?;` line handles the error